ffi/src/lib.rs

```diff
@@ -132,6 +132,14 @@ pub use utils::log;
 pub type BindingIterator = vec::IntoIter<Binding>;
 pub type BindingListIterator = std::slice::Chunks<'static, mentat::Binding>;
 
+/// Helper macro for asserting one or more pointers are not null at the same time.
+#[macro_export]
+macro_rules! assert_not_null {
+    ($($e:expr),+ $(,)*) => ($(
+        assert!(!$e.is_null(), concat!("Unexpected null pointer: ", stringify!($e)));
```

Comment:
`stringify!` arbitrary expression might lead to some odd reports, but I assume that `assert!` captures enough information (file/line) to be useful.